atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -42,6 +44,39 @@ def __init__(self, tasker, workflow, registries):
 
         self.registries = deepcopy(registries)
 
+    def find_push_conf_registry(self, registry):
+        """
+         Determine if we are on either an orchestrator node or a worker node.
+         From there, figure out where the information is.
+        """
+        if are_plugins_in_order(self.workflow.buildstep_plugins_conf, 'orchestrate_build'):
+            # orchestrator node
+            if not self.workflow.build_result.annotations:
+                return None
+
+            conf = {}
+            builds = self.workflow.build_result.annotations['worker-builds']
+
+            for plat, annotation in builds.items():
+                digests = json.loads(annotation['digests'])
+
+                for digest in digests:
+                    if digest['registry'] == registry:
+                        conf[digest['tag']] = ManifestDigest(v2=digest['digest'])
```

Comment:
I started thinking down that road and got distracted.  But yeah I agree with your thought process there. :-)  Let me poke at it.